crates/core/src/lib.rs

```diff
@@ -19,8 +19,10 @@ mod yarn_value;
 
 pub mod prelude {
     //! Types and functions used all throughout the runtime and compiler.
+    #[cfg(all(feature = "bevy", feature = "serde"))]
```

Comment:
```suggestion
    #[cfg(any(feature = "bevy", feature = "serde"))]
```

Reply:
Whoops, good spot!

Just occurred to me that I didn't mention why I made this change in the PR - I was getting an 'unused import' warning when building the `yarnspinner` crate on its own